README.md

```diff
@@ -539,7 +500,7 @@ In either case:
 
 ## Syntax
 
-* <a name="no-for"></a>Never use `for`, unless you know exactly why. Most of the 
+* <a name="no-for"></a>Never use `for`, unless you know exactly why. Most of the
```

Comment:
we could probably just do a blanket never use for. i've never once seen it in real ruby code